cypress/integration/tests/manage-germplasm/germplasm-details.feature

```diff
@@ -7,16 +7,38 @@ Feature: Germplasm Details
 
 @TestCaseKey=IBP-T3178
 @sanity-test
-    Scenario: Check pedigree tree
-        Given I am on the Germplasm Manager page
+@clean-install
+    Scenario: Check pedigree 
+        Given germplasm records already exists
+        And I am on the Manage Germplasm page of specified program
         When I click on the GID of a germplasm
         And I navigate to the Pedigree tab of germplasm details
         Then I should be able to see the pedigree tree
+@TestCaseKey=IBP-T3178
+@sanity-test
+@update-install
+    Scenario: Check pedigree 
+        Given I am on the Manage Germplasm page of specified program
+        When I click on the GID of a germplasm
+        And I navigate to the Pedigree tab of germplasm details
+        Then I should be able to see the pedigree tree        
+
+@TestCaseKey=IBP-T3180
+@sanity-test
+@clean-install
+    Scenario: Check pedigree 
```

Comment:
The last two scenarios should be "Check Pedigree Graph"

Reply:
Done